src/tests/clipperUI/components/previewViewer/augmentationPreview_tests.tsx

```diff
@@ -29,13 +25,64 @@ export class AugmentationPreviewTests extends TestModule {
 	}
 
 	protected tests() {
+		function isPositive(value: number): number {
+			if (value > 0) {
+				return 1;
+			} else {
+				return 0;
+			}
+		}
 		test("The tab order flow from the header to the preview title is correct in Augmentation mode", () => {
 			let mockClipperState = this.getMockAugmentationModeState();
 			let defaultComponent = <AugmentationPreview clipperState={mockClipperState} />;
 			MithrilUtils.mountToFixture(defaultComponent);
 
-			Assert.tabOrderIsIncremental([Constants.Ids.highlightButton, Constants.Ids.sansSerif, Constants.Ids.serif, Constants.Ids.decrementFontSize,
+			MithrilUtils.simulateAction(() => {
+				document.getElementById(Constants.Ids.serif).click();
+			});
+			Assert.tabOrderIsIncremental([Constants.Ids.highlightButton, Constants.Ids.serif, Constants.Ids.decrementFontSize,
 				Constants.Ids.incrementFontSize, Constants.Ids.previewHeaderInput]);
+
+			MithrilUtils.simulateAction(() => {
+				document.getElementById(Constants.Ids.sansSerif).click();
+			});
+			Assert.tabOrderIsIncremental([Constants.Ids.highlightButton, Constants.Ids.sansSerif, Constants.Ids.decrementFontSize,
+				Constants.Ids.incrementFontSize, Constants.Ids.previewHeaderInput]);
+		});
+
+		test("When a pair of codependent buttons have conditional attributes, they both respond with the same behavior when clicked and have different values from each other", () => {
+			let mockClipperState = this.getMockAugmentationModeState();
+			let defaultComponent = <AugmentationPreview clipperState={mockClipperState} />;
+			MithrilUtils.mountToFixture(defaultComponent);
+			let unSelectedTabShouldBeNegative = -1;
+
+			MithrilUtils.simulateAction(() => {
+				document.getElementById(Constants.Ids.serif).click();
+			});
+			let selectedTabIndices = [document.getElementById(Constants.Ids.serif).tabIndex];
+			let unselectedTabIndices = [document.getElementById(Constants.Ids.sansSerif).tabIndex];
+
+			MithrilUtils.simulateAction(() => {
+				document.getElementById(Constants.Ids.sansSerif).click();
+			});
+			selectedTabIndices.push(document.getElementById(Constants.Ids.sansSerif).tabIndex);
+			unselectedTabIndices.push(document.getElementById(Constants.Ids.serif).tabIndex);
+
+			strictEqual(selectedTabIndices[0], selectedTabIndices[1], "When the serif button is clicked the tabIndex should be the same as the tabIndex of the sansSerif button when clicked");
+			ok(isPositive(selectedTabIndices[0]), "Selected tabIndex should be greater than 0");
+
+			strictEqual(unselectedTabIndices[0], unselectedTabIndices[1], "When the serif button is not clicked the tabIndex should be the same as the tabIndex of the sansSerif button when not clicked");
+			strictEqual(unselectedTabIndices[0], unSelectedTabShouldBeNegative, "The unselected tabsIndices should be -1");
+
+			notStrictEqual(selectedTabIndices[0], unselectedTabIndices[0], "The buttons should have different values when one is clicked");
```

Comment:
This line is not necessary since we already asserted the first value is positive and the second value is negative